fusor-ember-cli/package.json

```diff
@@ -46,9 +46,6 @@
     "ember-disable-proxy-controllers": "^1.0.0",
     "ember-drag-drop": "0.3.1",
     "ember-export-application-global": "^1.0.3",
-    "ember-idx-button": "~0.1.3",
-    "ember-idx-forms": "0.5.1",
-    "ember-idx-modal": "~0.1.4",
```

Comment:
I just added this commit to remove the ember-idx components which are no longer needed after the OSP modals are refactored.